internal/config/config.go

```diff
@@ -67,6 +43,19 @@ var (
 	}
 )
 
+// View is a read-only view of the Open Match configuration.
+// New accessors from Viper should be added here.
+type View interface {
```

Comment:
Why are you removing view.go and the tests?

Reply:
we are not using `view.Sub` function anymore so basically view.go has only this structure defined. I remove view.go and the tests to consolidate the repo a bit.